src/tests/libecalc/core/consumers/test_direct_emitter.py

```diff
@@ -1,27 +1,48 @@
 from datetime import datetime
 
 import pytest
-from libecalc import dto
-from libecalc.core.consumers.venting_emitter import VentingEmitter
+from libecalc.common.units import Unit
```

Comment:
This test should probably be moved. presentation.yaml.test_venting_emitter.py?

Reply:
Moved